src/main/java/io/kestra/plugin/dbt/cloud/AbstractDbtCloud.java

```diff
@@ -46,12 +44,17 @@ public abstract class AbstractDbtCloud extends Task {
     @NotNull
     String token;
 
+    private static final Duration HTTP_READ_TIMEOUT = Duration.ofSeconds(3);
```

Comment:
```suggestion
    private static final Duration HTTP_READ_TIMEOUT = Duration.ofSeconds(60);
```

Reply:
you should raise the timeout because you are lowering it right now (default is 5s)

Reply:
Haha 🥇